app/v2/models/Binders.scala

```diff
@@ -25,8 +25,8 @@ object Binders {
 
     override def bind(key: String, value: String): Either[String, GuaranteeReferenceNumber] = value match {
       case grnPattern("GB", _) | grnPattern("XI", _) => Right(GuaranteeReferenceNumber(value))
-      case grnPattern(_, _)                          => Left("The guarantee reference number must be for a GB or XI guarantee.")
-      case _                                         => Left("The guarantee reference number is not in the correct format.")
+      case grnPattern(_, _)                          => Left("ERROR_INVALID_GRN_COUNTRY_CODE")
+      case _                                         => Left("ERROR_INVALID_GRN_FORMAT")
```

Comment:
I don't particularly like using strings like this, but I'm not sure we have a better option.

However, can we extract these strings to constants please? We should define them in only one place.